### ISSUE TITLE
example color clock with Error 500

### ISSUE THREAD

**Issue by pfpulux** [lona-web-org/lona#358]
If I run the example from the tutorial, I get this error:

```
LonaRuntimeWorker_0            ERROR    20:12:43.478093 lona.view_runtime Exception raised while running <__main__.Index object at 0x7f47d2854550>
  Traceback (most recent call last):
    File "/home/u/.local/lib/python3.11/site-packages/lona/view_runtime.py", line 345, in start
      return_value=self.view.handle_request(self.request),
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/home/u/play/lona/a4.py", line 71, in handle_request
      self.sleep(1)
    File "/home/u/.local/lib/python3.11/site-packages/lona/view.py", line 304, in sleep
      return self._await_sync(asyncio.sleep(delay, result))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/home/u/.local/lib/python3.11/site-packages/lona/view.py", line 101, in _await_sync
      ).result()
        ^^^^^^^^
    File "/usr/lib/python3.11/concurrent/futures/_base.py", line 456, in result
      return self.__get_result()
             ^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
      raise self._exception
    File "/home/pulux/.local/lib/python3.11/site-packages/lona/view.py", line 79, in await_awaitable
      finished, pending = await asyncio.wait(
                          ^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3.11/asyncio/tasks.py", line 415, in wait
      raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
  TypeError: Passing coroutines is forbidden, use tasks explicitly.
```
Insall via pip + system modules Python version 3.11.1.

**Comment by fscherf** [comment]
@pfpulux: thanks for reporting! This seems to be a python 3.11 specific problem in the `View.sleep()` implementation of Lona. Until this is fixed, you should be able to avoid this problem by using another python version, or by using `time.sleep()` for now

**Comment by fscherf** [comment]
@pfpulux: I created a PR. Could you verify if this PR fixes you problem?

**Comment by pfpulux** [comment]
how can I use this? The code from the example

```
class Index(View):
    def handle_request(self, request):
        timestamp = Strong()

        html = HTML(
            H1('Clock'),
            P(
                Span('The current time is: '),
                timestamp,
            ),
        )

        while True:

            # set current time
            timestamp.set_text(datetime.now())

            # show html
            self.show(html)

            # sleep for one second
            self.sleep(1)
```

produce the same error.

**Comment by fscherf** [comment]
@pfpulux: Did you try my PR? I agree, without my changes, running python3.11, your example does not work. When i install the new version using `pip install git+https://github.com/lona-web-org/lona@fscherf/fix-python311-asyncio-wait` it works.
Can you verify this, or give me some more information?

**Comment by pfpulux** [comment]
The test with this ^^^ (pip install) works.

**Comment by fscherf** [comment]
@pfpulux: Interesting. This seems to crash in aiohttp, not in Lona. On which platform are you running? From the traceback it looks like some kind of unix

**Comment by pfpulux** [comment]
voidlinux

**Comment by pfpulux** [comment]
`fix-python311-asyncio-wait` works.

**Comment by fscherf** [comment]
@pfpulux: The patch and the test are merged now. I will notify you here when the changes are released. Thanks for reporting and testing!

**Comment by fscherf** [comment]
@pfpulux: I released 1.12.4, which should fix this problem
